test/concurrency-master.spec.js

```diff
@@ -3,9 +3,20 @@ var Promise = require('bluebird');
 
 var ConcurrencyMaster = require('../lib/concurrency-master');
 
+var active = 0;
+var maxActive = 0;
 function wait(n) {
     return function() {
-        return Promise.delay(n);
+        active++;
+        if (active > maxActive) {
+            maxActive = active;
+        }
+        return new Promise(function(resolve, reject) {
+            setTimeout(resolve, n);
```

Comment:
Yes I did -- as I mentioned in the commit message https://github.com/juttle/concurrency-master/commit/e221a7804d7b51b449ffa29ac9c26ad0e6424ef6 I wanted to see if this worked with both the Promise implementation that comes with Node and fancy bluebird promises.